x/posts/internal/keeper/handler.go

```diff
@@ -281,3 +283,31 @@ func handleMsgAnswerPollPost(ctx sdk.Context, keeper Keeper, msg types.MsgAnswer
 	}
 	return &result, nil
 }
+
+func handleMsgRegisterReaction(ctx sdk.Context, keeper Keeper, msg types.MsgRegisterReaction) (*sdk.Result, error) {
+	_, isAlreadyRegistered := keeper.DoesReactionForShortCodeExist(ctx, msg.ShortCode, msg.Subspace)
+
+	if isAlreadyRegistered {
+		return nil, sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, fmt.Sprintf(
+			"reaction with shortcode %s and subspace %s has already been registered", msg.ShortCode, msg.Subspace))
+	}
```

Comment:
I would merge these lines by doing

```suggestion
	if _, isAlreadyRegistered := keeper.DoesReactionForShortCodeExist(ctx, msg.ShortCode, msg.Subspace); isAlreadyRegistered {
		return nil, sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, fmt.Sprintf(
			"reaction with shortcode %s and subspace %s has already been registered", msg.ShortCode, msg.Subspace))
	}
```